tests/foreman/longrun/test_provisioning_computeresource.py

```diff
@@ -144,8 +144,7 @@ def test_positive_provision_vmware_with_host_group(
             "start=1".format(VMWARE_CONSTANTS['cluster'], vmware.vmware_datacenter),
             'ip': None,
             'mac': None,
-            'interface': "compute_network={},"
-            "compute_type=VirtualVmxnet3".format(vmware.vmware_net_id),
+            'interface': f"compute_network={vmware.vmware_net_id}," "compute_type=VirtualVmxnet3",
```

Comment:
@shubhamsg199 @Gauravtalreja1 this looks good to me , need your second eye here.

Reply:
Yep, Looks good, anyways these long run tests have been migrated to `tests/foreman/cli/test_computeresource_vmware.py` with these changes